docs/modules/operation/pages/deep-dive/meta-data.adoc

```diff
@@ -254,3 +266,21 @@ The following {page-component-title} components allow the use of the Metadata DS
 * <<reference:configuration/ttl-rpc.adoc#metadata-ttls, Using metadata for TTLs>>.
 * <<deep-dive/thresholds/thresholding.adoc#ga-threshold-metadata, Expression-Based Thresholds>>.
 * <<deep-dive/notifications/introduction.adoc#ga-notifications, Notifications>>.
+* <<deep-dive/alarms/alarm-handling.adoc#ga-advanced-alarm-handling, Alarmd Drools rules>>.
+
+Furthermore, some {page-component-title} configuration files allow the use of Metadata DSL to access the global scope `scv` in order to reference credentials in the Secure Credentials Vault:
+
```

Comment:
Are you sure all below configs parse metadata. Poller configs, collector configs I can see that they are supported. Snmp configs I have done it myself. Other configs we have to validate.

Reply:
Yeah, you bring up a good point Chandra. I rushed this thinking that all the SCV meta data interpolation stuff would be in this release and it's not. Dino sent me some of the NMS issues where those features were put into place in H33. Which wouldn't be in Meridian 2023. 

I'm going to close this issue and re-open it for Foundation 2024 instead.